src/labels/CustomLabels.labels

```diff
@@ -2637,18 +2668,46 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>WARNING: There are records saved to this batch. Any changes you make won&apos;t affect existing records. We will not attempt to match existing records using updated donation matching rule settings, unless those records are edited.</value>
     </labels>
     <labels>
-        <fullName>bgeBatchTemplateSetFieldOptions</fullName>
+        <fullName>bgeFormMatchingModalText</fullName>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Header for Batch Template Field Options</shortDescription>
-        <value>Set Field Options</value>
+        <shortDescription>Header text to explain the options on the page</shortDescription>
```

Comment:
Specify which page? We might as well make the description text user friendly instead of relying on the label name for context